physics.hpp

```diff
@@ -71,5 +79,25 @@ float phy::vec3::DotProduct(vec3 const object)
 	float result = res.x + res.y + res.z;
 	return result;
 }
+vec3 phy::vec3::CrossProduct(vec3 const object)
+{
+	vec3 res;
+	res.x = (y*object.z) - (z*object.y);
+	res.y = (z*object.x) - (x*object.z);
+	res.z = (x*object.y) - (y*object.x);
+	return res;
+}
+vec3 phy::PhysicsObj::TotalForce(const vector<vec3> object )
```

Comment:
Make another function to impart a force to this object from another object.
To calculate total force all you have to do is just go through the forces vector in the current object itself, you don't need any arguments.

Reply:
wait i didnt get it. do you want me to create total force as a data member of the vec3 class so i can keep incrementing it for each object

Reply:
There's a vector<vec3> member in PhysObj called forces. Go through each element of this vector to find the total force. When an another object wants to apply a force to this object, it will call the applyForce function (yet to be written) with the force to be applied as the parameter. In the applyForce function, the new force is appended to the forces vector. That's how I imagined it to be, if you have a different way then it's up to you to choose whichever

Reply:
i had a similar idea and i implemented the function you are talking about in the first line